<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HscRewriteCurve</class>
 <widget class="QWidget" name="HscRewriteCurve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="curveLayout">
        <item row="0" column="0">
         <widget class="QCustomPlot" name="plot" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>状态</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>重写模次</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>21</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="PlotControlsBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>绘制控制</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_17">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="plotControlsLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="pointsLabel">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>显示点数</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinPoints">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>5000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="labelYStep">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Y 刻度</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinYStep">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Y 最小值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinAxesMin">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>-65536</number>
              </property>
              <property name="maximum">
               <number>65536</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Y 最大值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinAxesMax">
              <property name="minimumSize">
               <size>
                <width>69</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>69</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>-65536</number>
              </property>
              <property name="maximum">
               <number>65536</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_AutoScale">
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>8</height>
             </size>
            </property>
            <property name="text">
             <string>Y轴自动缩放</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_Channels">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoScroll">
             <bool>false</bool>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectItems</enum>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideMiddle</enum>
            </property>
            <property name="resizeMode">
             <enum>QListView::Adjust</enum>
            </property>
            <property name="modelColumn">
             <number>0</number>
            </property>
            <property name="selectionRectVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ResetVisible">
            <property name="text">
             <string>曲线显示</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="savePNGButton">
            <property name="text">
             <string>存储为PNG</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
